src/pynwb/file.py

```diff
@@ -1150,16 +1150,6 @@ def _tablefunc(table_name, description, columns):
     return t
 
 
-def ElectrodeTable(name='electrodes',
```

Comment:
This might break downstream code. Hopefully not because people should not be using this function, but it is not conventionally marked as private with leading underscores, so they might. What do you think about changing this to something like:

```python
def ElectrodeTable(name='electrodes', 
                   description='metadata about extracellular electrodes'):
    warn("The ElectrodeTable convenience function is deprecated. Please create a new instance of the ElectrodesTable class instead.", DeprecationWarning)
    return ElectrodesTable()
```